runtime/consts.go

```diff
@@ -20,28 +20,4 @@ const (
 	// ReconcileAtAnnotation is the annotation used for triggering a
 	// reconciliation outside of the defined schedule.
 	ReconcileAtAnnotation string = "fluxcd.io/reconcileAt"
```

Comment:
Why didn't this get removed here?

Reply:
I can't introduce the dependency (including the replace) without Go Modules complaining about a non-existing version.

Reply:
Lol, of course, it's the three step shuffle :-)